1 Table.sql

```diff
@@ -0,0 +1,17 @@
+Use PetDB
+GO
+
+Drop table if exists Pet 
+GO
+create table dbo.Pet(
```

Comment:
The table name should be customer since it is to keep record of the customers and their pets.

Reply:
Fixed